rogue-thi-app/components/RoomMap.jsx

```diff
@@ -352,7 +392,7 @@ export default function RoomMap ({ highlight, roomData }) {
               <LayersControl.BaseLayer
                 key={floorName + (searchText || 'empty-search')}
                 name={translateFloors(floorName)}
-                checked={i === filteredFloorOrder.length - 2}
+                checked={i === filteredFloorOrder.length - (searchText ? 1 : 2)}
```

Comment:
When searching for `I` this shows the basement by default. 
However basements are not that relevant, which is why we should still display the ground floor.